### PR TITLE
add skeleton sessionmap page

### PR DESCRIPTION
**Pull request by nbilenko**
## Description
Issue: https://github.com/openoakland/woeip/issues/64

Skeleton sessionmap page (according to wireframe 9 in https://trello.com/c/f81CfYp8/35-document-v2-ui-improvements-for-implementation), with some placeholder properties (marked with TODOs) and placeholder map section.

### CODE REVIEW

File: woeip/templates/session.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
+{% extends '__base.html' %}
+{% block content %}
+{% load tz %}
+{% timezone "America/Los_Angeles" %}
+<!-- TODO: move all style to static files -->
+<style>
```
Comment (TangoYankee) [inline woeip/templates/session.html]: CSS Should be in a dedicated file

**Review by TangoYankee — CHANGES_REQUESTED** [review]
Fix linting issues and pass Travis checks, [before requesting code review](https://github.com/openoakland/woeip/blob/master/.github/contributing.md#pull-requests). Create issues for any `TODO` or `unused import` that you would like to track.

Yeah, they're just linting issues. But, if we start choosing to ignore small failures, we might end up missing a big failure.

**Comment by nbilenko** [comment]
> Fix linting issues and pass Travis checks, [before requesting code review](https://github.com/openoakland/woeip/blob/master/.github/contributing.md#pull-requests). Create issues for any `TODO` or `unused import` that you would like to track.
> 
> Yeah, they're just linting issues. But, if we start choosing to ignore small failures, we might end up missing a big failure.

Yes I realize I should've run `make quality` before submitting, sorry about that.
Just to clarify, are we not allowing any `TODO` in the code? Currently the linting fails if there are any. This isn't ideal for WIP pull requests.

**Comment by TangoYankee** [comment]
> Just to clarify, are we not allowing any `TODO` in the code? Currently the linting fails if there are any. This isn't ideal for WIP pull requests.

If we've planned the code in the `Issue`, there shouldn't be a need for `TODO` in a Pull Request. If you'd like to argue in favor of allowing `TODO`, open an `Issue`.

**Comment by nbilenko** [comment]
> > Just to clarify, are we not allowing any `TODO` in the code? Currently the linting fails if there are any. This isn't ideal for WIP pull requests.
> 
> If we've planned the code in the `Issue`, there shouldn't be a need for `TODO` in a Pull Request. If you'd like to argue in favor of allowing `TODO`, open an `Issue`.

Ok, I can remove TODOs - I was just confirming that's the intended workflow. I was using TODOs to flag the placeholder parts of the code to be fleshed out as part of the same issue which wouldn't really warrant new issues but are dependent on other work (e.g. separating out static files, since it seems we haven't established a method for handling those yet; or querying for device name since it wasn't clear how we plan to distinguish the air quality one). But I can just keep track of these items via comments.

**Comment by theecrit** [comment]
Blocked until associated issue is more fully defined.

**Comment by theecrit** [comment]
Blocked by #64 requiring greater detail.

**Comment by nbilenko** [comment]
Closing due to move to REST framework.

**Pull request closed without merging**